### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by HakanEgeOzerol**
Added comments to css, changed px values to vw, minor adjustments to picture sizes.

### CODE REVIEW

**Comment by HakanEgeOzerol** [comment]
Added comments to css, changed px values to vw, minor adjustments to picture sizes.

File: projectstyle.css [hunk 1]
```diff
@@ -36,12 +36,14 @@ body {
```
Comment (AlvesBernardo) [inline projectstyle.css:36]: you are still calling Bannertitle here it should be bannerTitle no?

File: projectstyle.css [hunk 2]
```diff
@@ -36,12 +36,14 @@ body {
...
     text-align: center;
-    color: #1162AD ;
+    color: #1162AD;
     font-size: 4em !important;
 }
+
 /*-------------------------*/
 /*Style for the pictures and upper text*/
 /*-------------------------*/
+
```
Comment (AlvesBernardo) [inline projectstyle.css:46]: same should it not be picture?

File: projectstyle.css [hunk 3]
```diff
@@ -68,10 +70,12 @@ body {
 #Rightpicture > img {
```
Comment (AlvesBernardo) [inline projectstyle.css:70]: same here with the Rightpicture

File: projectstyle.css [hunk 4]
```diff
@@ -88,17 +92,18 @@ body {
...
 /*-------------------------*/
-#Bottomtext{
+
+#Bottomtext {
     grid-column: 1/7;
     padding-right: 3vw;
     padding-left: 3vw;
     margin-bottom: 2vw;
 }
-
```
Comment (AlvesBernardo) [inline projectstyle.css]: go over with the names on the css first letter should not be big

**Review by AlvesBernardo — CHANGES_REQUESTED** [review]

**Review by AlvesBernardo — APPROVED** [review]

**Pull request merged**